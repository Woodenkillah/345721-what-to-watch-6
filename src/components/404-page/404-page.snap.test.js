import React from 'react';
import {render} from '@testing-library/react';
import {Router} from 'react-router-dom';
import {createMemoryHistory} from 'history';
import Page404 from './404-page';

it(`Should render correctly`, () => {
  const history = createMemoryHistory();
  const {container} = render(
      <Router history={history}>
        <Page404/>
      </Router>
  );
  expect(container).toMatchSnapshot();
});
