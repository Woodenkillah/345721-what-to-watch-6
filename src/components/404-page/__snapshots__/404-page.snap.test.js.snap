// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correctly 1`] = `
<div>
  <div
    class="user-page"
  >
    <header
      class="page-header user-page__head"
    >
      <div
        class="logo"
      >
        <a
          aria-current="page"
          class="logo__link active"
          href="/"
        >
          <span
            class="logo__letter logo__letter--1"
          >
            W
          </span>
          <span
            class="logo__letter logo__letter--2"
          >
            T
          </span>
          <span
            class="logo__letter logo__letter--3"
          >
            W
          </span>
        </a>
      </div>
      <h1
        class="page-title user-page__title"
      >
        404 - Page is not found!
      </h1>
      <p
        style="color: rgb(201, 179, 126); position: absolute; left: 200px; right: 200px; text-align: center; top: 75%;"
      >
        Please check the correctness of URL or contact to an administrator, whoever he is...
      </p>
      <a
        href="/"
        style="color: rgb(201, 179, 126); position: absolute; left: 200px; right: 200px; text-align: center; bottom: -100px; cursor: pointer; font-size: 24px;"
      >
        Return to home page
      </a>
    </header>
  </div>
</div>
`;
